src/lib/FreeC/Environment/LookupOrFail.hs

```diff
@@ -44,6 +45,20 @@ lookupIdentOrFail srcSpan scope name = do
   entry <- lookupEntryOrFail srcSpan scope name
   return (entryIdent entry)
 
+-- | Looks up the Agda identifier for a Haskell function, (type)
+--   constructor or (type) variable with the given name or reports a fatal
+--   error message if the identifier has not been defined.
+--
+--   If an error is reported, it points to the given source span.
+lookupAgdaIdentOrFail
+  :: SrcSpan  -- ^ The source location where the identifier is requested.
+  -> IR.Scope    -- ^ The scope to look the identifier up in.
+  -> IR.QName -- ^ The Haskell identifier to look up.
```

Comment:
Why doesn't brittany want to indent the comment? : /

```suggestion
  :: SrcSpan  -- ^ The source location where the identifier is requested.
  -> IR.Scope -- ^ The scope to look the identifier up in.
  -> IR.QName -- ^ The Haskell identifier to look up.
```

Reply:
Brittany doesn't format comments. They have to be aliged manually.

Reply:
What a pity (for @JonasHoefer  ; ))!